recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ test:
     - pytools/test
   commands:
     - pip check
-    - cd pytools/test && python -m pytest
+    - pip install -e . && python -m pytest
```

Comment:
How come the `pip install` is necessary?

Reply:
I don't know, this is the only workaround I found. Without this, the tests fail with:

```
>       assert (FairRibbon().tag_name
                == FairRibbon().tag_name
                == DottedName(("pytools", "test", "test_pytools", "FairRibbon")))
E       AssertionError: assert DottedName('test', 'test_pytools', 'FairRibbon') == DottedName('pytools', 'test', 'test_pytools', 'FairRibbon')
E        +  where DottedName('test', 'test_pytools', 'FairRibbon') = test_tag.<locals>.FairRibbon().tag_name
E        +    where test_tag.<locals>.FairRibbon() = <class 'test.test_pytools.test_tag.<locals>.FairRibbon'>()
E        +  and   DottedName('pytools', 'test', 'test_pytools', 'FairRibbon') = <class 'pytools.tag.DottedName'>(('pytools', 'test', 'test_pytools', 'FairRibbon'))
```

I don't fully understand the impact of this change: https://github.com/inducer/pytools/commit/ba4769f28415877188c9d3a8335297960f7db0d5

Reply:
- No need to preserve the tests, they're installed as part of the package.
- Run `python -m pytest pytools`